database/mysql/0013_user_management.sql

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ * Copyright (C) 2017-2018 Eaton
+ *
+ * This software is confidential and licensed under Eaton Proprietary License
+ * (EPL or EULA).
+ *
+ * This software is not authorized to be used, duplicated or disclosed to
+ * anyone without the prior written permission of Eaton.
+ * Limitations, restrictions and exclusions of the Eaton applicable standard
+ * terms and conditions, such as its EPL and EULA, apply.
+ *
+ */
+
+/*
+ * @file    0013_user_management.sql
+ * @author: <a href="mailto:XavierMillieret@Eaton.com">Xavier, Millieret (E9902068)</a>
+ * @brief   User management data base definition. 
+ */
+
+
+/* For details on schema version support see the main initdb.sql */
+SET @bios_db_schema_version = '201802140001';
+SET @bios_db_schema_filename = '0013_user_management.sql';
+
+use box_utf8;
+
+/* This should be the first action in the SQL file */
+START TRANSACTION;
+INSERT INTO t_bios_schema_version (tag, timestamp, filename, version) VALUES('begin-import', UTC_TIMESTAMP() + 0, @bios_db_schema_filename, @bios_db_schema_version);
+/* Report the value */
+SELECT * FROM t_bios_schema_version WHERE tag = 'begin-import' order by id desc limit 1;
+COMMIT;
+
+/*
+*
+* User management tables creations 
+*
+*
+*/
+
+/* User table */
+CREATE TABLE IF NOT EXISTS t_user(
+    uid             INTEGER UNSIGNED  NOT NULL,
+    PRIMARY KEY (uid)
+);
+
+/* Vcard table */
+CREATE TABLE IF NOT EXISTS t_vcard(
```

Comment:
The "vCard" is a specific IT-industry term for a specific type and markup of data : https://en.wikipedia.org/wiki/VCard 

This table does not seem to implement it, I think: it has the `id_vcard` as a key for each single couple of keytag:value, so the id would be unique if I read it right. 

If the idea is to store components of a real vCard line by line, maybe this table should be split into two: one for grabbing a unique id_vcard (bound to a uid), and another for id_vcard:keytag:value - a lot of entries with same id_vcard.

Reply:
I totaly agree with your comment, so I sugest to call this table t_um_entity_attibute_value, I modify my PR in this way